packages/react/src/themes/teams-dark/components/Dropdown/dropdownVariables.ts

```diff
@@ -0,0 +1,22 @@
+import { DropdownVariables } from '../../../teams/components/Dropdown/dropdownVariables'
+
+export default (siteVars): Partial<DropdownVariables> => ({
+  backgroundColor: siteVars.colors.grey[650],
+  backgroundColorHover: siteVars.colors.grey[550],
+  borderColor: 'transparent',
+  borderColorFocus: siteVars.colors.brand[400],
+  color: siteVars.colors.grey[250],
+  selectedItemColor: siteVars.colors.grey[250],
+  listBoxShadow: `0 .2rem .6rem 0 rgba(0, 0, 0, 0.25)`,
```

Comment:
Can we add the box shadow value to siteVariables?

Reply:
Used siteVariables.shadowLevel3, which is added in dark theme as well: `export const shadowLevel3 = '0 .8rem 1rem -0.2rem rgba(0, 0, 0, .25)'` In high contrast there is no shadow on the list